modules/deploy/pages/deployment-option/cloud/security/cloud-authentication.adoc

```diff
@@ -1,16 +1,43 @@
 = Authentication
-:description: Learn about Redpanda Cloud authentication and authentication services.
+:description: Learn about Redpanda Cloud authentication.
 :page-cloud: true
 :page-categories: Management, Security
 
 Redpanda Cloud ensures the highest level of authentication for both users and services.
 
 == User authentication
 
-Redpanda Cloud authenticates users directly using their email and password.
+Redpanda provides user authentication with email/password and single sign-on. 
+
+=== Email
+
+Redpanda Cloud authenticates data plane users with emails and passwords.
 Passwords are hashed (a one-way function that makes the original value unrecoverable,
 and effectively encrypted) and salted at rest using https://en.wikipedia.org/wiki/Bcrypt[bcrypt^].
 
+=== Single sign-on
+
+You can configure single sign-on (SSO) with an OIDC-based identity provider (IdP) for control plane authentication to Dedicated or BYOC clusters. Redpanda integrates with any OIDC-compliant IdP that supports discovery, including Okta, Auth0, Microsoft Entra, and Active Directory Federation Services (AD-FS). After SSO is enabled for an organization, all new users in the organization are authenticated using SSO. 
+
+==== Integrate IdP
+
+On the *Users* page, you can add connections for up to two different IdPs. Enter the client ID, client secret, and discovery URI for the IdP. (See your IdP documentation for these values. The discovery URI may be called the well known URL or the `issuer_url`.) By default, the connection is added in a disabled state. Edit the connection to enable it, and optionally assign a read, write, or admin role to users who log in with the IdP. 
```

Comment:
You must integrate your IdP with Redpanda Cloud to use SSO.
^^Add as first sentence?